examples/clinical_pipeline/chat_project.json

```diff
@@ -0,0 +1,683 @@
+{
```

Comment:
This looks like the Stave config file that's generated in Stave when the file is uploaded. Must of the content of this file is duplicate with the `base_ontology` file. Do we actually need to keep the whole thing here. @Piyush13y what do you think

Reply:
@hunterhector Both the stave json files are actually a combination of `base_ontology` and `clinical_ontology` from when they were created. This is important because stave doesn't support multiple ontology files yet, and we have to provide it all the annotations/ontologies for it to work for medical scenario. That's what's being fed to stave through these combined ontology files.

Reply:
Can we actually use code to generate these instead of hard-code them? For example, if there are some updates in the ontology file then this file also needs to be changed.

Reply:
Yeah, good idea. I think we can write a small script to just combine both our `base_ontology` and `clinical_ontology` json files, which can then be picked up by this example to do what it does.

Reply:
If you are looking to merge two ontology files, we have the scripts:
https://asyml-forte.readthedocs.io/en/latest/toc/ontology_generation.html  check for "-m" option.

If `forte` is installed, we should have the `generate_ontology` script installed too. You can also directly call it from python

Reply:
I have a simple question here, since it is a Stave config file (consists of ontology and config two parts), is it feasible to use the merge function in ontology generation? Maybe I should first merge the two ontology files, and then, merge the ontology and config to a Stave config file?

Reply:
Yes you are right, here is some concept clarification:
1. Our ontology generation script can create merged ontology files only, since it is a Forte tool
2. Stave can be considered as an extension to Forte. The Stave config file uses the Forte ontology, but what it creates is not technically an ontology file anymore, as you said, it creates only a config. For example, the config shows whether to visualize each type of annotation, these functionalities are out of the scope of ontology files.